README.md

```diff
@@ -1,3 +1,5 @@
+![GitHub branch checks state](https://img.shields.io/github/checks-status/drachenfrucht1/godotcord/master) ![GitHub repo size](https://img.shields.io/github/repo-size/drachenfrucht1/godotcord)
```

Comment:
https://img.shields.io/github/checks-status/drachenfrucht1/godotcord/master somehow only shows checks:pending when I test it (although all checks passed). https://img.shields.io/github/workflow/status/Drachenfrucht1/godotcord/Compile/master shows build:passing (could also use Compile for Donwload or both)